sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/NotificationHookLiveTests.cs

```diff
@@ -345,13 +345,12 @@ public async Task UpdateWebNotificationHookWithEveryMember()
         [RecordedTest]
         [TestCase(true)]
         [TestCase(false)]
-        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/18004")]
         public async Task GetHooksWithMinimumSetup(bool useTokenCredential)
         {
             MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient(useTokenCredential);
 
             string hookName = Recording.GenerateAlphaNumericId("hook");
-            var hookToCreate = new WebNotificationHook(hookName, new Uri("http://contoso.com/"));
+            var hookToCreate = new EmailNotificationHook(hookName) { EmailsToAlert = { "fake@email.com" } };
```

Comment:
It doesn't really matter here if we have a `WebNotificationHook` or an `EmailNotificationHook`, as long as we have a hook. We can't really test web hooks right now (https://github.com/Azure/azure-sdk-for-net/issues/21504), so updated this one.